dom/src/main/scala/fs2/dom/Window.scala

```diff
@@ -67,7 +70,25 @@ object Window {
 
       implicit def given_Dom_F: Dom[F] = Dom.forAsync
 
-      def document: HtmlDocument[F] = window.document.asInstanceOf[HtmlDocument[F]]
+      def document: HtmlDocument[F] =
+        window.document.asInstanceOf[HtmlDocument[F]]
+
+      def domContentLoaded: F[Unit] = F.async_[Unit] { cb =>
```

Comment:
This is not cancelable—instead we can use `asyncCheckAttempt` with an `AbortController` to make it cancelable.